test/RoadRegistry.Tests/BackOffice/Scenarios/ModifyRoadNodeScenarios.cs

```diff
@@ -1166,7 +1166,8 @@ public Task when_modifying_a_start_node_connecting_two_segments_as_a_fake_node_b
             Segment2Added.Status = Segment1Added.Status;
             Segment2Added.Morphology = Segment1Added.Morphology;
             Segment2Added.Category = Segment1Added.Category;
-            Segment2Added.MaintenanceAuthority = Segment1Added.MaintenanceAuthority;
+            Segment2Added.MaintenanceAuthority.Code = Segment1Added.MaintenanceAuthority.Code;
```

Comment:
Another case of sporadic test failure - cruel punishment for copying references :-)